eng/common/pipelines/templates/archetype-typespec-emitter.yml

```diff
@@ -202,16 +202,19 @@ extends:
                 Write-Host "Using overrides.json to generate emitter-package.json"
                 tsp-client generate-config-files `
                 --package-json '$(buildArtifactsPath)/lock-files/package.json' `
-                --output-dir '$(Build.ArtifactStagingDirectory)' `
-                --emitter-package-json-path  '${{ parameters.EmitterPackageJsonPath }}' `
-                --overrides '$(buildArtifactsPath)/overrides.json'
+                --output-dir '$(Build.SourcesDirectory)' `
+                --emitter-package-json-path "$(Build.SourcesDirectory)/${{ parameters.EmitterPackageJsonPath }}" `
```

Comment:
I would require parameters.EmitterPackageJsonPath be a rooted path, or have `tsp-client` resolve it using working directory.   joining the string here will always pass  `--emitter-package-json-path "$(Build.SourcesDirectory)"` when the parameter is empty.   Best to default it at the top to `$(Build.SourcesDirectory)/eng/emitter-package.json` and let users override it as needed.